### DISCUSSION TITLE
How to migrate Mail from EOS 5 to 6

### DISCUSSION THREAD

**Discussion by frank-wo** [elementary/mail#689] [Q&A]
i know this is not an issue but a question ( i have no idea how to get in contact with EOS support):

Since Mail is broken in EOS 5 (#677) I'm looking for a solution to get my message base and email addresses back.

My first question is: is it possible to upgrade the old Mail app to the new one, but on EOS 5.1.7 - without installing EOS 6.

My second question: where all email data has been saved on Mail in EOS 5.1.7., how can I save this data when migrating to EOS 6.

TIA
Frank

**Comment by danirabbit** [comment]
Hey Frank,

It is technically possible to use the Mail Flatpak to install the new Mail on OS 5, but Mail now uses online accounts from System Settings that are not compatible with the old version. So it's probably not worth trying to pursue this route.

I'm not sure what you mean by "email data". All of your messages are stored on the email server that provides your account. Migrating to another email client only requires logging in to your email provider.
